test-suite/schema-generation/allowed-values/allowed-values-basic_test_valid_PASS.json

```diff
@@ -3,16 +3,21 @@
         "odd-flag": 11,
         "field-dates": [
             "2020-01-01",
-            "2020-02-29"
+            "2020-02-29",
+            "2020-07-04"
         ],
         "children": [
             {
-                "even-flag": 10,
+                "even-flag": 4,
                 "field-date": "2020-01-01"
             },
             {
-                "even-flag": 2,
+                "even-flag": 10,
                 "field-date": "2020-02-29"
+            },
+            {
+                "even-flag": 2,
+                "field-date": "2020-07-04"
```

Comment:
Minor, non-blocking: I just want to make sure I understand: these changes are minor and unrelated to the testing of the PR, correct? I didn't know if there was some relation in the test suite to these changes around Markdown/CommonMark conformance?

Reply:
Yes. This was just a unit test improvement to add some diversity in the test vector values..